src/main/java/com/atlassian/bitbucket/jenkins/internal/status/BuildStatusPoster.java

```diff
@@ -0,0 +1,55 @@
+package com.atlassian.bitbucket.jenkins.internal.status;
```

Comment:
I think this class needs some tests, 
Some cases (but not all cases) to test are:
What happens there is no `BitbucketRevisionAction`?
What happens if there is no server configured (or it has been deleted)